examples/hello-world/app.py

```diff
@@ -1,23 +1,30 @@
 import os
-from opentelemetry import trace
+from opentelemetry import trace, baggage, metrics
+from opentelemetry.context import attach, detach
 from honeycomb.opentelemetry import configure_opentelemetry, HoneycombOptions
 
 configure_opentelemetry(
     HoneycombOptions(
         debug=True, # prints exported traces & metrics to the console, useful for debugging and setting up
         apikey=os.getenv("HONEYCOMB_API_KEY"), # Honeycomb API Key, required to send data to Honeycomb
         service_name="otel-python-example", # Dataset that will be populated with data from this service in Honeycomb
-        enable_local_visualizations=True, # Will print a link to a trace produced in Honeycomb to the console, useful for debugging
+        # enable_local_visualizations=True, # Will print a link to a trace produced in Honeycomb to the console, useful for debugging
         # traces_apikey = None, Set a specific Honeycomb API key just for traces
         # metrics_apikey = None, Set a specific Honeycomb API key just for metrics
         # service_version = None, Set a version for this service, will show up as an attribute on all spans
+        endpoint=os.getenv("HONEYCOMB_API_ENDPOINT",
```

Comment:
Another note, is there a reason in particular we're using this `HONEYCOMB_API_ENDPOINT` as an environment variable here and in the docker-compose and not `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT` or `OTEL_EXPORTER_OTLP_METRICS_ENDPOINT`?

It works fine here in the configure_opentelemetry function but I don't think it goes through our options parsing if we use it as an ENV VAR with `opentelemetry-instrument` 

If HONEYCOMB_API_ENDPOINT is a spac that supposed to work with both ways we'll need to develop more on our options parsing.

Reply:
Opened #98 for this! Good to go ✨